compliance/compliance.go

```diff
@@ -268,9 +318,19 @@ func (c *Compliance) runRecv(ctx context.Context, client sensor.ComplianceServic
 		case *sensor.MsgToCompliance_Ack:
 			switch t.Ack.GetAction() {
 			case sensor.MsgToCompliance_NodeInventoryACK_ACK:
-				c.umh.HandleACK()
+				switch t.Ack.GetMessageType() {
```

Comment:
should we add defaults or is it ok to just ignore it?

Reply:
For now there are two message types only. Defaults would be useful when we add third type but forgot to update the switch statements. Maybe a default with some loud error would prevent such problems, I'll add it.